back/engines/commercial/analysis/app/controllers/analysis/web_api/v1/analyses_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
           authorize ::Analysis::Analysis
           @analyses = policy_scope(::Analysis::Analysis).order(created_at: :asc)
           @analyses = paginate(@analyses)
-          @analyses = @analyses.includes(:custom_fields)
+          @analyses = @analyses.includes(:main_custom_field, :additional_custom_fields)
```

Comment:
I think we still need `all_custom_fields` in here also, which would be the equivalent of the previous `custom_fields`